app/presenters/tree_builder_chargeback_rates.rb

```diff
@@ -35,6 +35,24 @@ def x_get_tree_roots(count_only, options)
         end
         objects
       end
+    when :cb_rates
+      # the rate accordion merge the compute rate and the storage rate in one unic rate grouping by description
+      rates = ChargebackRate.all
+      grouped_rates = rates.group_by(&:description)
+      if count_only
+        grouped_rates.length
+      else
+        objects = []
+        grouped_rates.sort.map do |description_group|
+          objects.push(
+            # We identified by the structure Compute_id:Storage_id
+            :id    => description_group[1][0].id.to_s + ":" + description_group[1][1].id.to_s,
```

Comment:
@amaurygonzalez it blows up here if description_group[1] only has 1 element in array

[----] I, [2016-03-17T12:09:48.853478 #10380:416c2b4]  INFO -- : Started GET "/chargeback/explorer" for 127.0.0.1 at 2016-03-17 12:09:48 -0400
[----] I, [2016-03-17T12:09:48.923544 #10380:416c2b4]  INFO -- : Processing by ChargebackController#explorer as HTML
[----] F, [2016-03-17T12:09:49.636005 #10380:416c2b4] FATAL -- : Error caught: [NoMethodError] undefined method `id' for nil:NilClass
/home/hkataria/dev/manageiq/app/presenters/tree_builder_chargeback_rates.rb:49:in `block in x_get_tree_roots'
/home/hkataria/dev/manageiq/app/presenters/tree_builder_chargeback_rates.rb:46:in `map'
/home/hkataria/dev/manageiq/app/presenters/tree_builder_chargeback_rates.rb:46:in `x_get_tree_roots'
/home/hkataria/dev/manageiq/app/presenters/tree_builder.rb:340:in `x_get_tree_objects'
/home/hkataria/dev/manageiq/app/presenters/tree_builder.rb:306:in `x_build_dynatree'
/home/hkataria/dev/manageiq/app/presenters/tree_builder.rb:245:in `build_tree'
/home/hkataria/dev/manageiq/app/presenters/tree_builder.rb:173:in `initialize'
/home/hkataria/dev/manageiq/app/controllers/application_controller/feature.rb:28:in `new'
/home/hkataria/dev/manageiq/app/controllers/application_controller/feature.rb:28:in `build_tree'